src/App.js

```diff
@@ -21,31 +21,50 @@ function useApp() {
   return { centralTeams, pacificTeams };
 }
 
+function SortTeamsByRank(teams, year) {
```

Comment:
もしただの関数なら JS 的には camelCase の方がより一般的です

```suggestion
function sortTeamsByRank(teams, year) {
```

ただ、関数名と実体に乖離がある（ソートだけでなく抽出も行っている）ので、より正確な関数名にするか、関数を2つに分ける方が誤解が生じづらいかなと思いました

Reply:
関数を2つに分けるようにしました！ 85b17ec69d74a0940ae1683a0d9537a5e5218337